src/Topologies/dtopology2d.jl

```diff
@@ -84,6 +84,30 @@ struct DistributedTopology2D{
 
     "a NamedTuple of vectors of `(e,face)` "
     boundaries::BF
+    "internal local elements (lidx)"
+    internal_elems::Vector{Int}
+    "local elements (lidx) located on process boundary"
+    perimeter_elems::Vector{Int}
+    "total number of unique non-boundary vertices"
+    nglobalvertices::Int
+    "total number of unique non-boundary faces"
+    nglobalfaces::Int
+    "global id for each unique ghost vertex"
+    ghost_vertex_gidx::Vector{Int}
+    "global id for each unique ghost face"
+    ghost_face_gidx::Vector{Int}
+    "number of unique vertices to be communicated to neighboring processes"
+    comm_vertex_lengths::Vector{Int}
+    "number of unique faces to be communicated to each neighboring process"
+    comm_face_lengths::Vector{Int}
+    "neighbor process locations in neighbor_pids for each of the ghost vertices"
+    ghost_vertex_neighbor_loc::Vector{Int}
+    "offset array for `ghost_vertex_neighbor_loc`"
+    ghost_vertex_comm_idx_offset::Vector{Int}
```

Comment:
a ragged array representation:
for `i = 1:length(ghost_vertex_gidx)`, we have that for `j = ghost_vertex_comm_idx_offset[i]:ghost_vertex_comm_idx_offset[i+1]-1`
the ghost vertex `ghost_vertex_gidx[i]` should get sent to `neighbor_pids[ghost_vertex_neighbor_loc[j]]`.

Reply:
include this in the docstring